src/collection.js

```diff
@@ -158,24 +160,26 @@ export default class Collection {
    * Adds a record to the local database.
    *
    * Options:
-   * - {Boolean} synced     Sets record status to "synced" (default: false).
-   * - {Boolean} forceUUID  Enforces record creation using any provided UUID
-   *                        (default: false).
+   * - {Boolean} synced   Sets record status to "synced" (default: false).
+   * - {Boolean} forceId  Forces the id field from the record to be used, instead
+   *                      of one that is generated automatically (default: false).
```

Comment:
Makes me think that for creations, we could raise if `forceId` is `false` and the method receives a record object containing an id; right now the error is silenced.